integration/makefile.mk

```diff
@@ -0,0 +1,24 @@
+GOFAIL_BINARY = $(shell pwd)/gofail
+
+# run all integration tests
+.PHONY: run-all-integration-tests
+run-all-integration-tests: run-integration-test-sleep
+
+# run integration test - sleep
+.PHONY: run-integration-test-sleep
+run-integration-test-sleep: build-gofail execute-integration-test-sleep remove-gofail
+
+.PHONY: execute-integration-test-sleep
+execute-integration-test-sleep:
+	@echo "GOFAIL_BINARY = ${GOFAIL_BINARY}"
+	cd ./integration/sleep/failpoints && ${GOFAIL_BINARY} enable && cd .. && go build -o integration_test_sleep . && cd failpoints && ${GOFAIL_BINARY} disable
+	cd ./integration/sleep && ./integration_test_sleep && rm integration_test_sleep
```

Comment:
If the execution fails, the clean-up will not happen.

I have introduced another target `clean-all-integration-tests` where the user can manually clean up the binaries left behind though, do you think this will be enough?